syncopy/tests/backend/test_timefreq.py

```diff
@@ -153,10 +153,10 @@ def test_mtmconvol():
     # -------------------------
 
     taper = 'dpss'
-    taperopt = {'Kmax' : 10, 'NW' : 2}
+    taper_opt = {'Kmax' : 10, 'NW' : 2}
```

Comment:
I can't comment on the code above - there's another `plot` assert (L111)

Reply:
Same on L181

Reply:
The plot on L201 will go unnoticed during headless `pytest` runs - is it kept in here intentionally for debugging purposes in case anything goes wrong and we have to do some digging?

Reply:
More plots are on L230, L244, L259, L304, L319, L366, L382, debug prints are on L267 and L327

Reply:
OK I'll remove the debug prints! The plots are nice when trying to actually see what was tested/the result. But you say there is no point to `assert` them?!